how-to/customize-workspace/client/src/integrations-shapes.ts

```diff
@@ -6,12 +6,12 @@ import type {
 	HomeSearchResult
 } from "@openfin/workspace";
 import type { BrowserWindowModule, Page } from "@openfin/workspace-platform";
-import type { LoggerCreator } from "./logger-shapes";
+import type { ModuleDefinition, ModuleImplementation, ModuleList } from "./module-shapes";
 
 /**
  * Integration manager provides environment methods and data.
```

Comment:
Update comment to say integration helpers?

Reply:
Done